src/pages/SignUpPage/SignUpPage.styles.ts

```diff
@@ -0,0 +1,258 @@
+  LOGO_TOP: 36.54,
+  EMAIL_INPUT_TOP: 122.42,
+  USERNAME_INPUT_TOP: 225.65,
+  PASSWORD_INPUT_TOP: 328.88,
+  BUTTON_TOP: 432.12,
+  LOGIN_LINK_TOP: 517.08,
+  SIGNUP_WITH_TOP: 570.98,
+  ICONS_TOP: 609.35,
+  LEGAL_TEXT_TOP: 689.74,
+
+  INPUT_LEFT: 41.11,
+  INPUT_WIDTH: 411.1,
+  INPUT_HEIGHT: 81.31,
+  BUTTON_HEIGHT: 62.54,
+  LOGO_WIDTH: 266.14,
+  LOGO_HEIGHT: 45.68,
+  LOGO_LEFT: 111.45,
+  ICON_SIZE: 42.02,
+  ICON_SEPARATION: 54, // Separation (space) between icons
+  GOOGLE_POS_X: 119.68,
+  LOGIN_LINK_LEFT: 399,
+  LEGAL_TEXT_WIDTH: 405.62,
+};
+
+const percentW = (px: number) => (px / GEOMETRY_PX.PAGE_WIDTH) * 100 + '%';
+
+const percentH = (px: number) => (px / GEOMETRY_PX.PAGE_HEIGHT) * 100 + '%';
+
+const fontVw = (px: number, capPx: number, theme: Theme) => {
+  const vw = (px / GEOMETRY_PX.PAGE_WIDTH) * 100;
+
+  const rem = theme.typography.pxToRem(capPx);
+
+  const dummy = theme.palette.primary.main;
+  return {
+    fontSize: `clamp(8px, ${vw}vw, ${rem})`,
+
+    'data-theme': dummy,
+  };
+};
+
+// Use theme to avoid lint error
+const fontVwThemeUsage = (theme: Theme) => {
+  const dummy = theme.palette.primary.main;
+  return dummy;
+};
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+fontVwThemeUsage;
+
+const fontVwUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+fontVwUsage;
+
+const dummyThemeUsage2 = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+dummyThemeUsage2;
+
+const themeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+themeUsage;
+
+const useTheme = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+useTheme;
+
+const dummy = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+dummy;
+
+const finalThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+finalThemeUsage;
+
+const lastThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+lastThemeUsage;
+
+const ultimateThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+ultimateThemeUsage;
+
+const finalUltimateThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+finalUltimateThemeUsage;
+
+const dummyThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+dummyThemeUsage;
+
+export const getPageContainerStyles = (theme: Theme) => ({
+  position: 'relative',
+  width: '100%',
+
+  maxWidth: `${GEOMETRY_PX.PAGE_WIDTH}px`,
```

Comment:
maxWidth is defined using a hardcoded px string: `${GEOMETRY_PX.PAGE_WIDTH}px`. The project guideline forbids hardcoded px values. Consider using responsive breakpoints, theme spacing, or keep the geometry in numbers with clear documentation. If a px-based max width is truly required to match a design artifact, add a comment/justification and centralize constants in theme or a constants file.

**Sugerencia:**
```typescript
Either derive the value from theme (theme.breakpoints, theme.spacing) or document/centralize the px value. Avoid scattered hardcoded px strings.
```